admin/deletePlace.js

```diff
@@ -0,0 +1,75 @@
+import {
+    collection,
+    onSnapshot,
+    doc,
+    deleteDoc,
+} from 'https://www.gstatic.com/firebasejs/10.12.1/firebase-firestore.js';
+import db from '../firebase/db.js';
+
+const placesCollectionRef = collection(db, 'places');
+
+const placesContainer = document.getElementById('places');
+
+async function handleDelete(id) {
+    const confirmDelete = confirm("Are you sure you want to delete this place?")
+    if (confirmDelete) {
+        try {
+            const docRef = doc(db, 'places', id);
+            await deleteDoc(docRef);
+            alert('Successfully deleted place.');
+        } catch (error) {
+            console.error(error);
+            alert(error.message);
+        }
+    } else {
+        return;
+    }
+}
```

Comment:
palagay nalang siguro to sa `editPlace.js` nga code baii, para di mag double initialization ang mga stuffff, kay same lang siya sa editPlace nga codee

so i add lang siya sa `editPlace.js` tapos i add lang yan ang `handleDelete`